indexing-service/src/main/java/org/apache/druid/indexing/common/task/HadoopIndexTask.java

```diff
@@ -94,10 +94,10 @@
 
 public class HadoopIndexTask extends HadoopTask implements ChatHandler
 {
+  public static final String TYPE = "index_hadoop";
```

Comment:
Same here regarding `getType()` using `TYPE`.

Reply:
good catch and fixed.